external/llvh/lib/Support/Unix/Program.inc

```diff
@@ -200,7 +200,11 @@ static bool Execute(ProcessInfo &PI, StringRef Program,
   // If this OS has posix_spawn and there is no memory limit being implied, use
   // posix_spawn.  It is more efficient than fork/exec.
 #ifdef HAVE_POSIX_SPAWN
+#if defined(TARGET_OS_TV) && TARGET_OS_TV
```

Comment:
What is the issue being worked around here?

Reply:
Apple TV needs to use posix_spawn (does not have fork()) but MemoryLimit did not exist for Apple TV (at least at the time I was working on this patch).

Reply:
What do you mean? MemoryLimit is a parameter to this function.

Reply:
Sorry I didn't say that correctly. Apple TV needs to use this path even if MemoryLimit is not 0.